src/lang/en.json

```diff
@@ -110,6 +110,7 @@
   "header_heroes": "Heroes",
   "header_blog": "Blog",
   "header_ingame": "Ingame",
+  "header_player_search": "Player Search",
```

Comment:
only the english language file change is required.  The rest will be done via crowdin